Contest_API.md

```diff
@@ -1337,13 +1440,20 @@ zip archive. These must be stored directly from the root of the zip
 file, i.e. there must not be extra directories (or files) added unless
 these are explicitly part of the submission content.
 
-#### POST and PUT submissions
+#### Submission capabilities 
+
+To add a submission clients can use the `POST` method on the submissions endpoint or the
+`PUT` method directly on an object url. One of the following [capabilities](#capabilities)
+is required to add submissions:
 
-To add a submission one can use the `POST` method on the submissions endpoint or the
-`PUT` method directly on an object url. `POST` is typically used by teams submitting
-to the contest and `PUT` is used by admin users or tools.
-Both must include a valid JSON object with the same properties the submission
-endpoint returns with a `GET` request with the following exceptions:
+| Name              | Description
+| :---------------- | :----------
+| team_submit       | POST a submissision as a team
+| proxy_submit      | POST a submission as a proxy (able to submit on behalf of a team)
```

Comment:
For clarity, I'd propose
```suggestion
| proxy_submit      | POST a submission as a proxy (able to submit on behalf of any team)
```

Reply:
This would precludes a proxy (i.e. an account) that has the right to submit for some, but not all, teams. This would make sense e.g. in the original use case we imagined for proxies, i.e. multi-site contests.

Reply:
Agreed. I hope neither of you want to add API for the proxy to tell which team(s) it can submit for? :)

In practise I think it's likely that we wouldn't have to restrict a proxy to a subset of teams b/c only the teams it can submit for will have either network or authentication access to it, but I agree we should't preclude this in the API. How about "on behalf of team(s)"? That implies it may be more than 1 but doesn't imply all teams.

Reply:
I was pushing another commit so I went with "team(s)", let me know if it works.

Reply:
> hope neither of you want to add API for the proxy to tell which team(s) it can submit for? :)

Nope :).